packages/config/src/Config.ts

```diff
@@ -121,6 +123,20 @@ export function getConfig(projectRoot: string, options: GetConfigOptions = {}):
       dynamicConfigPath: paths.dynamicConfigPath,
       staticConfigPath: paths.staticConfigPath,
     };
+
+    if (options.omitPrivateExpoConfig) {
```

Comment:
```suggestion
    if (options.isPublicManifest) {
```
Maybe something like this indicating what the config is being used for.

Reply:
would like to avoid the word "manifest" because we're trying to start being more intentional about separating this from the app config. Would `isPublicConfig` or `isPublicExpoConfig` be ok with you?